de_docs/getting-started/user-guides/agenda-voting.md

```diff
@@ -0,0 +1,140 @@
+
+Finally, you'll need to learn how to set the `votechoice` for your tickets in order to cast a "Yes", "No", or "Abstain" vote for an agenda. By default, your tickets will cast "Abstain" votes. If you are solo-staking with `dcrwallet`, go here. If you are pool-staking with `dcrwallet`, `Paymetheus`, or `Decrediton`, go here.
+
+---
+
+## **Solo-voting**
+
+Through the command-line, you'll want to familiarize yourself with the `dcrctl --wallet getvotechoices` and `dcrctl --wallet setvotechoice "agendaid" "choiceid"` commands.
+
+The first command, `dcrctl --wallet getvotechoices`, returns JSON resembling this:
+
+```
+{
+  "version": 4,
+  "choices": [
+    {
+      "agendaid": "sdiffalgorithm",
+      "agendadescription": "Change stake difficulty algorithm as defined in DCP0001",
+      "choiceid": "abstain",
+      "choicedescription": "abstain voting for change"
+    },
+    {
+      "agendaid": "lnsupport",
+      "agendadescription": "Request developers begin work on Lightning Network (LN) integration",
+      "choiceid": "abstain",
+      "choicedescription": "abstain from voting"
+    }
+  ]
+}
+```
+
+The second command, `dcrctl --wallet setvotechoice "agendaid" "choiceid"`, let's you set your votechoice. `"agendaid"` is found via the `getvotechoices` command above, and `"choiceid"` can be `yes`, `no`, or `abstain`.
+
+For example, issuing `dcrctl --wallet setvotechoice sdiffalgorithm yes` results in the following changes to `dcrctl --wallet getvotechoices`:
+
+```
+{
+  "version": 4,
+  "choices": [
+    {
+      "agendaid": "sdiffalgorithm",
+      "agendadescription": "Change stake difficulty algorithm as defined in DCP0001",
+      "choiceid": "yes",
+      "choicedescription": "change to the new algorithm"
+    },
+    {
+      "agendaid": "lnsupport",
+      "agendadescription": "Request developers begin work on Lightning Network (LN) integration",
+      "choiceid": "abstain",
+      "choicedescription": "abstain from voting"
+    }
+  ]
+}
+```
+
+---
+
+## **Stakepool-voting**
+
+If your Stakepool has updated to the latest stakepool software, you will find a "Voting" page in the navigation menu with dropdown options for each agenda. After you've chosen how you want your tickets to vote, simply press the "Update Voting Preferences" to save your votechoices. Below you'll find an image of the votechoices for vote version 4.
+
+<img src="/img/stakepool-voting-page.png">
+
+---
+
+## **Verifying Your Vote**
+
+The easiest method to track your how your tickets actually voted is to use the [block explorer](https://mainnet.decred.org) to view each of your votes.
+
+The block explorer has been updated to display YES and NO votes for each agenda. If a ticket voted abstain, it will not display anything. The first ticket in this example image voted "ABSTAIN" for both agendas, the second ticket voted "YES" for both agendas, and the third ticket voted "YES" for the lnsupport agenda and "ABSTAIN" for the sdiffalgorithm agenda.
```

Comment:
The bit about the block explorer not showing abstain is no longer true.  It shows `YES`, `NO`, or `ABSTAIN` now for both agendas for all v4 votes.  This also means the image needs to be updated.